### ISSUE TITLE
reactor basics howto is lacking important information

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2722]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#2722|
|Type|defect|
|Created|2007-06-25 12:03:40Z|
|Branch|https://github.com/twisted/twisted/tree/reactor-basics-2722|

Like, what is an event loop? How does the first paragraph end? Where are the interfaces for the reactor kept (more relevant for PDF version than HTML version)?

<details><summary>Searchable metadata</summary>

```
trac-id__2722 2722
type__defect defect
reporter__itamarst itamarst
priority__normal normal
milestone__ 
branch__branches_reactor_basics_2722 branches/reactor-basics-2722
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1182773020000000 1182773020000000
changetime__1247423532000000 1247423532000000
version__None None
owner__ 
cc__thijs cc__mwh
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

Replying to [itamarst](#2722):
> Like, what is an event loop? How does the first paragraph end?

Yea, there's some stuff missing there, it ends with:

```
This HOWTO introduces the Twisted reactor, describes the basics of the
reactor and links to 
```

> Where are the interfaces for the reactor kept (more relevant for PDF version than HTML version)?

I guess you want to see a reference to the full path here like ```twisted.internet.interfaces.IReactorTCP```?

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

(In [27103]) Branching to 'reactor-basics-2722'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

r27105 is up for review.

**Comment by twisted-trac** [comment]
|[<img alt="mwhudson's avatar" src="https://avatars.githubusercontent.com/u/672022?s=50" width="50" height="50">](https://github.com/mwhudson)<a name="note_6"></a>|@mwhudson set owner to @thijstriemstra|
|-|-|

I'm obviously in a picky mood tonight!

The changes look good, perhaps apart from the word "polling", which might give the wrong impression.  I think you can just change it to "calling".  If you agree with me, please change it and land the branch.

Thanks for all these branches :)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [27126]) Merge reactor-basics-2722: Finish the first paragraph of the howto and explain what an event loop is.

Author: thijs
Reviewer: mwh
Fixes: [#2722](https://github.com/twisted/twisted/issues/2722)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
